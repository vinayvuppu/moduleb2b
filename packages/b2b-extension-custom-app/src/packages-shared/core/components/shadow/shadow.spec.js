import React from 'react';
import { shallow } from 'enzyme';
import Shadow from './shadow';

const createTestProps = custom => ({
  children: <span>{'foo'}</span>,
  depth: '1',
  className: 'foo-style',
  ...custom,
});

describe('rendering', () => {
  it('outputs correct tree', () => {
    const props = createTestProps();
    const wrapper = shallow(<Shadow {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
