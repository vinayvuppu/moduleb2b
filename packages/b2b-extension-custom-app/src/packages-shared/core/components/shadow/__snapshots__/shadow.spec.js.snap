// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering outputs correct tree 1`] = `
<div
  className="shadow-1 foo-style"
>
  <span>
    foo
  </span>
</div>
`;
